packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeProvider.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Generator.CSharp.ClientModel.Providers
     /// <summary>
     /// This class provides the set of serialization models, methods, and interfaces for a given model.
     /// </summary>
-    internal sealed class MrwSerializationTypeProvider : TypeProvider
+    public sealed class MrwSerializationTypeProvider : TypeProvider
```

Comment:
does this need to be public?

Reply:
Yes, because it is used in ClientModelPlugin

Reply:
Reverting now that we have collapsed back to single dll.